packages/api/cms-api/src/dam/files/files.service.ts

```diff
@@ -271,7 +271,7 @@ export class FilesService {
                         ? {
                               width: image.width,
                               height: image.height,
-                              exif: await exifr.parse(file.path),
+                              exif: await exifr.parse(file.path).catch(() => undefined),
```

Comment:
Using this approach you are silencing _all_ errors, even valid ones. Wouldn't it be better to only parse the EXIF data if the file is a supported file format? See compatibility table here: https://www.npmjs.com/package/exifr

Reply:
> Using this approach you are silencing all errors, even valid ones.

I argue that an exif parsing error is never important enough to prevent the file upload.

Exif data is auto-generated. The user (typically) doesn't change it manually and (typically) doesn't really care about it. Blocking the upload because of an exif parsing error is probably just annoying to the user.

Therefore, I don't think it matters if all errors are silenced or not.

Reply:
I just discovered in the docs that the `silentErrors` option is enabled by default. Meaning all valid errors are silent anyway.

Only wrong arguments or an unknown file format triggers an exception.

see https://www.npmjs.com/package/exifr#user-content-optionssilenterrors